bindgen-cli/options.rs

```diff
@@ -353,6 +353,18 @@ struct BindgenCommand {
     /// Derive custom traits on a `union`. The <CUSTOM> value must be of the shape <REGEX>=<DERIVE> where <DERIVE> is a coma-separated list of derive macros.
     #[arg(long, value_name = "CUSTOM")]
     with_derive_custom_union: Vec<String>,
+    /// Generate extern wrappers for inlined functions
+    #[arg(long, requires = "experimental")]
+    wrap_non_extern_fns: bool,
+    /// Sets the path for any extra files must be created due to the presence of inlined functions.
+    #[arg(long, requires = "experimental", value_name = "PATH")]
+    wrap_non_extern_fns_path: Option<String>,
```

Comment:
I wonder if we can just use `PathBuf` here, would save some conversions elsewhere.